app/static/js/createEvents.js

```diff
@@ -168,5 +168,8 @@ $(document).ready(function() {
      $(this).closest("tr").remove();
   });
 
+  $('#deleteEventModal').on('click', function() {
+      $("#deleteWarning").modal("show")
+  })
```

Comment:
I think you can have a button trigger a modal automatically with something bootstrappy in the html. That lets us avoid a little javascript

Reply:
did the bootsrappy thing